conclave-common/src/main/kotlin/com/r3/conclave/common/internal/EnclaveCallType.kt

```diff
@@ -0,0 +1,28 @@
+package com.r3.conclave.common.internal
+
+/**
+ * Enumeration of all possible host -> enclave calls
+ */
+enum class EnclaveCallType {
+    INITIALISE_ENCLAVE,
+    START_ENCLAVE,
+    STOP_ENCLAVE,
+    GET_ENCLAVE_INSTANCE_INFO_QUOTE,
+    GET_KDS_PERSISTENCE_KEY_SPEC,
+    SET_KDS_PERSISTENCE_KEY,
+    CALL_MESSAGE_HANDLER;
+
+    fun toByte(): Byte {
+        check(VALUES.size < Byte.MAX_VALUE)
```

Comment:
A bit unnecessary I think. Better to document clearly instead.

Reply:
I've removed the runtime check and added a unit test just to be thorough.